run.py

```diff
@@ -46,7 +46,7 @@ def run(
             experiment,
             data={Const.input_col: data[Const.input_col]},
             labels=data[Const.label_col],
-            plugins=logger_plugins,
+            plugins=logger_plugins + [OutputAnalyserPlugin()],
```

Comment:
looks good! but maybe let's not add it as a default for now? I'll need to comment this out all the time otherwise :)

Reply:
anyway, I'll comment it out later!

Reply:
Default plugins are defined in ``runner.py``, under ``obligatory_plugins``. So this is where we currently define optional plugins!